### ISSUE TITLE
Check for existing group upon creation

### ISSUE THREAD

**Issue by Samaritan1011001** [Samaritan1011001/Trovami#30]
Check if a group with the same members already exist for the user. If present, do not create a new one.

**Comment by alankley** [comment]
Hmm.  So if 9 of 10 are the same it's ok?
Maybe this check is not necessary?
Maybe user will eventually remove/delete members so they aren't identical.  Should we second guess them?
As long as only 1 group at a time is used, probably no harm in duplicate groups? As long as name/id is not the same

**Comment by Samaritan1011001** [comment]
Yes, but I was thinking we keep it simple in the beginning so it's easy
for us to get the base app running. I feel once we have the basic group
location sharing set up, we can add editing the group and it's members. It
will just let us concentrate on the more important aspect of live location
sharing without having to make it available for varying members of the
group. We have plenty on our hands already and extra features must be
considered after we have the base setup. What do you think?

**Comment by Samaritan1011001** [comment]
But then again we would need to put work into checking if a group with the
same members exists. So, I guess a duplicate group is less work. I'll put
that card to backlog for now and take it up later.

**Comment by alankley** [comment]
Editing the group is not currently supported?
I would think this is very basic to support from the beginning.
Groups with duplicate members don't do any harm. Adding checks to check for identical members is adding code.
Adding the check, however, is a small issue.  I don't think it's necessary but I don't feel strongly about it.

**Comment by Samaritan1011001** [comment]
Haha yeah, I realized. Let's go with duplicates. Do you want to move these
conversations to slack?

**Comment by Samaritan1011001** [comment]
Hey Alan,

I added cards that I think we can start with. The cards under "To Do" are
in order of priority that I think we need to use. If you have other
thoughts, happy to discuss.
The ones in "Backlog" are the ones I feel we can consider at a later date
when we have the base app refactored and running. Let me know what you
think.
Also, consider moving our conversations to slack or other instant messaging
platforms. It's just quicker and easier to communicate. Thanks.

**Comment by alankley** [comment]
I agree. I suggested the same on a previous email.
I have 2 different company Slack accounts.
I just created a new Slack account using alankley@gmail.com if you use
Slack.

**Comment by alankley** [comment]
I tried chatting through Gmail and it took me to Hangouts which is going
away.
Do you use Slack?  I couldn't find a Free version.
I have 2 company accounts with Slack.

I also use GroupMe Chat app

**Comment by Samaritan1011001** [comment]
Hi Alan,
I just created a slack workspace for trovami and added you. There is a free
Slack version. You should be able to log in to the workspace from the
invite I sent you.

**Comment by alankley** [comment]
Hello.
Are you still interested in keeping this project?
Did you receive warnings from Google about a Maps Key being exposed to the
public?

-Alan
